lib/version.ts

```diff
@@ -0,0 +1,2 @@
+import packageJson from '../package.json';
```

Comment:
🤔 I'm not sure that the package.json can be imported so with ESM (in case of real version).

Reply:
a test like this works
```
import packageJson from '../../package.json'

describe('Test version JSON', () => {
    it('test name', () => {
        expect(packageJson.name).toEqual('snyk-broker')
    })
})```